chainindex/gc.go

```diff
@@ -0,0 +1,113 @@
+package chainindex
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+)
+
+var (
+	log             = logging.Logger("chainindex")
+	cleanupInterval = time.Duration(8) * time.Hour
```

Comment:
Should we consider making this relate more closely to to `gcRetentionEpochs`? If you set that quite low then maybe this should reduce to be some portion of it. Something like min of 8 hours or `gcRetentionEpochs/4`.

Reply:
@rvagg Can we punt on this ? We now GC ALL tipsets(reverted and non-reverted) before a user configured epoch every 8 hours and I think that's good enough to start with.